### PR TITLE
[Cu f94xbn] create main layout

### PR DESCRIPTION
**Pull request by Buglik**
Main layout with two tabs( tab1 and tab2 for now) and avatar button with dropdown menu (logout option and some setting that will be provided later).

### CODE REVIEW

**Comment by Zummek** [comment]
Task linked: [CU-f94xbn Create main layout](https://app.clickup.com/t/f94xbn)

File: web-app/src/components/HeaderMenuElement.vue [hunk 1]
```diff
@@ -1,14 +1,11 @@
 <template>
   <q-item clickable tag="a" target="_blank" :href="link">
-    <q-item-section v-if="icon" avatar>
+    <q-item-section v-if="icon" class="lt-md" avatar>
```
Comment (Zummek) [inline web-app/src/components/HeaderMenuElement.vue]: Why do You want to show icon only on lower size screen than `md`?

File: web-app/src/components/HeaderMenuElement.vue [hunk 2]
```diff
@@ -19,7 +16,6 @@ import { Vue, Component, Prop } from 'vue-property-decorator'
 @Component
 export default class EssentialLink extends Vue {
```
Comment (Zummek) [inline web-app/src/components/HeaderMenuElement.vue]: Change class name to this same as the file name.

File: web-app/src/components/HeaderMenuElement.vue [hunk 3]
```diff
@@ -19,7 +16,6 @@ import { Vue, Component, Prop } from 'vue-property-decorator'
...
   @Prop({ type: String, required: true }) readonly title!: string
```
Comment (Zummek) [inline web-app/src/components/HeaderMenuElement.vue]: `label` sounds better than `title`

File: web-app/src/layouts/MainLayout.vue [hunk 4]
```diff
@@ -1,41 +1,121 @@
...
+                      color="primary"
+                      label="Logout"
+                      push
+                      size="sm"
+                      v-close-popup
+                    />
+                  </div>
+                </div>
+                <q-separator horizontal class="q-mx-xs" />
+                <div class="row  q-pa-md">
```
Comment (Zummek) [inline web-app/src/layouts/MainLayout.vue:96]:
```suggestion
                <div class="row q-pa-md">
```

File: web-app/src/layouts/MainLayout.vue [hunk 5]
```diff
@@ -44,60 +124,26 @@
...
 ];
 
 import { Vue, Component } from 'vue-property-decorator';
 
 @Component({
-  components: { EssentialLink },
+  components: { HeaderMenuElement },
 })
 export default class MainLayout extends Vue {
   leftDrawerOpen = false;
```
Comment (Zummek) [inline web-app/src/layouts/MainLayout.vue]: It's useless variable

File: web-app/src/layouts/MainLayout.vue [hunk 6]
```diff
@@ -44,60 +124,26 @@
...
+    title: 'Tab1',
+    link: '#',
   },
   {
-    title: 'Quasar Awesome',
-    caption: 'Community Quasar projects',
-    icon: 'favorite',
-    link: 'https://awesome.quasar.dev',
+    title: 'Tab2',
+    link: '#',
```
Comment (Zummek) [inline web-app/src/layouts/MainLayout.vue]: I think we can change tab titles to `News`, `Tournaments`

**Review by Zummek — COMMENTED** [review]
When You create/modify new view paste here screenshot

File: web-app/src/layouts/MainLayout.vue [hunk 7]
```diff
@@ -1,41 +1,121 @@
...
+        </div>
+
+        <q-toolbar-title class="lt-sm">
+          <q-icon name="emoji_events" />
+          Best tournament app
+        </q-toolbar-title>
+        <q-toolbar-title class="gt-xs text-center">
+          <q-icon name="emoji_events" />
+          Best tournament app
         </q-toolbar-title>
```
Comment (eastemployeer) [inline web-app/src/layouts/MainLayout.vue]:
```suggestion
        <q-toolbar-title :class="$q.screen.gt.xs ? 'text-center gt-xs' : ''">
          <q-icon name="emoji_events" />
          Best tournament app
        </q-toolbar-title>
```

File: web-app/src/layouts/MainLayout.vue [hunk 8]
```diff
@@ -1,41 +1,121 @@
...
 
-        <div>Quasar v{{ $q.version }}</div>
+        <div class="row col-3 gt-xs justify-end">
+          <div>
```
Comment (eastemployeer) [inline web-app/src/layouts/MainLayout.vue]: Useless div

File: web-app/src/layouts/MainLayout.vue [hunk 9]
```diff
@@ -1,41 +1,121 @@
...
-          round
-          icon="menu"
-          aria-label="Menu"
-          @click="leftDrawerOpen = !leftDrawerOpen"
-        />
-
-        <q-toolbar-title>
-          Quasar App
+        <div class="row col-3 gt-xs">
+          <HeaderMenuElement
```
Comment (eastemployeer) [inline web-app/src/layouts/MainLayout.vue]: External component for adding buttons with icons to the header? Really?

**Review by eastemployeer — CHANGES_REQUESTED** [review]
Code can be improved and simplified in several places.

File: web-app/src/components/HeaderMenuElement.vue (hunk 1 above)
Comment (Buglik) [reply web-app/src/components/HeaderMenuElement.vue]: that's some trash from sidebar option, removing whole HeaderMenuElement anyway

File: web-app/src/layouts/MainLayout.vue (hunk 5 above)
Comment (Buglik) [reply web-app/src/layouts/MainLayout.vue]: again some trash from sidebar version

**Review by Zummek — APPROVED** [review]
LGTM

**Review by eastemployeer — APPROVED** [review]

**Pull request merged**